oat-sa/extension-tao-itemqti
# Fix/AUT-3785/broken pci authoring

### Related to: https://oat-sa.atlassian.net/browse/AUT-3785

Also fixes:
- https://oat-sa.atlassian.net/browse/PISA25-753
- https://oat-sa.atlassian.net/browse/PISA25-751

### Summary

Prevent some styling and behavior conflicts between PCI and Item Authoring.

### Details

- break the grid layout control over PCI nested markup
- enforce the modal style in the item authoring so that PCI would not conflict

### How to test

- check out the branch: `git checkout -t origin/fix/AUT-3785/broken-PCI-authoring`
- open the backoffice
- import the items linked in the ticket
- open the items
- check the console: no error should arise
- the save button should be available
- new interactions should be draggable and droppable
- modal dialogs should be broken
